Author: Christoph Biedl <debian.axhn@manchmal.in-ulm.de>
Date: 2014-03-05
Description:
 Replace 8-bit characters in CDF summary data with spaces.
 .
 At the moment, uninitialized values are printed, resulting in
 differing results upon each call.
 .
 Present spaces instead until upstream provides a better solution.

--- a/src/readcdf.c
+++ b/src/readcdf.c
@@ -141,8 +141,9 @@
                                     j++, s += k) {
                                         if (*s == '\0')
                                                 break;
-                                        if (isprint((unsigned char)*s))
-                                                vbuf[j] = *s;
+                                        vbuf[j] = 
+                                                isprint((unsigned char)*s) ?
+                                                *s : ' ';
                                 }
                                 if (j == sizeof(vbuf))
                                         --j;
